### DISCUSSION TITLE
Rename commands name and description depending on user's locale

### DISCUSSION THREAD

**Discussion by jbaumgarten** [Rapptz/discord.py#9010] [Q&A]
Hi guys,

Is there any way to change commands name and description depending on the user's locale?

**Comment by jbaumgarten** [comment]
Regarding the discord documentation there are name_localizations and description_localizations used for that (https://discord.com/developers/docs/interactions/application-commands)

Any plan to implement it?

**Comment by Rapptz** [comment]
Localisation is implemented using [`locale_str`](https://discordpy.readthedocs.io/en/stable/interactions/api.html#discord.app_commands.locale_str) and [`Translator`](https://discordpy.readthedocs.io/en/stable/interactions/api.html#discord.app_commands.Translator). There's no example in the repository for this right now but the documentation walks you through the steps. Note that for simple strings you don't need to wrap them in `locale_str` since `auto_locale_strings` is set to `True` by default.

**Reply by jbaumgarten** [reply]
Ok thx. I'll try to find how to use it :)

**Reply by jbaumgarten** [reply]
> Localisation is implemented using [`locale_str`](https://discordpy.readthedocs.io/en/stable/interactions/api.html#discord.app_commands.locale_str) and [`Translator`](https://discordpy.readthedocs.io/en/stable/interactions/api.html#discord.app_commands.Translator). There's no example in the repository for this right now but the documentation walks you through the steps. Note that for simple strings you don't need to wrap them in `locale_str` since `auto_locale_strings` is set to `True` by default.

I succeed to make it work. Thx
However, I have a question. Is it possible to use Translator on View/Modals/Buttons/... objects?
